clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -97,8 +102,20 @@ public FileOutStream(long fileId, OutStreamOptions options) throws IOException {
     }
     mClosed = false;
     mCanceled = false;
-    mHostname = options.getHostname();
     mShouldCacheCurrentBlock = mTachyonStorageType.isStore();
+
+    // use local first as default policy
+    Class<? extends FileWriteLocationPolicy> locationPolicyClass = LocalFirstPolicy.class;
```

Comment:
This default should be a config parameter, so it is easy to change with something like Spark java opts.